include/swift/AST/ASTPrinter.h

```diff
@@ -217,6 +219,14 @@ class ASTPrinter {
     *this << Suffix;
   }
 
+  void printIntroducerKeyword(StringRef name,
+                              StringRef Suffix = " ") {
```

Comment:
I think we should keep the default `Suffix` the same between `printKeyword` and `printIntroducerKeyword`, otherwise it's confusing.